### PR TITLE
Octave array support in python_copy_vars_to.m

### PR DESCRIPTION
**Pull request by latot**
Fixes https://github.com/cbm755/octsympy/issues/335

### CODE REVIEW

**Comment by latot** [comment]
things

**Comment by latot** [comment]
okidoki, fixed

**Comment by cbm755** [comment]
ok, at least it doesn't just error out now.

But *again*, what is this for?

**Comment by latot** [comment]
mm, basically if we need it, i'mnot exactly an python user, but can help you, i think have more resources can help.

File: inst/private/python_copy_vars_to.m [hunk 1]
```diff
@@ -90,6 +90,10 @@
       c = length(a);
       c=c+1; a{c} = sprintf('%s%s.append(dict(zip(%s,%s)))', sp, in, inkeys, invalues);
 
+    elseif ismatrix(x)
+
+      c=c+1; a{c} = sprintf('%s%s.append(%s)', sp, in, sprintf(octave_array_to_python(x)));
+
```
Comment (cbm755) [inline inst/private/python_copy_vars_to.m]: how about `elseif (ismatrix(x) && (isnumeric(x) || islogical(x)))`?

**Comment by cbm755** [comment]
ok I guess this doesn't do any harm and its private..., change the above and I'll merge it.

**Comment by latot** [comment]
okidoki

**Pull request merged**